src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -4638,6 +4678,215 @@ test_decryption_events_case4 (void *unused)
    decryption_events_fixture_destroy (def);
 }
 
+static void
+_test_rewrap_with_separate_client_encryption (const char *src_provider,
+                                              const char *dst_provider)
+{
+   mongoc_uri_t *const uri = test_framework_get_uri ();
+   mongoc_client_encryption_opts_t *const ce_opts =
+      mongoc_client_encryption_opts_new ();
+   mongoc_client_t *const src_client =
+      test_framework_client_new_from_uri (uri, NULL);
+   mongoc_client_t *const dst_client =
+      test_framework_client_new_from_uri (uri, NULL);
```

Comment:
```suggestion
      test_framework_client_new_from_uri (uri, NULL);
   test_framework_set_ssl_opts (src_client);
   test_framework_set_ssl_opts (dst_client);
```

This may resolve SSL related failures in Evergreen.